### ISSUE TITLE
New source for pre-built images?

### ISSUE THREAD

**Issue by Gunnar-GO** [tsol/Open-PoniX#2]
At the moment t-sol.ru isn't aviable, so sadly the linked source for pre-built images isn't aviable.

i would like to test these variations, they sounds promissing.

**Comment by tsol** [comment]
Yes, the company was closed in 2016. We had to take down site several years after. I think server was re-used somehow and im not sure i can find anything now.

If any of our old users have those packets downloaded - i would appreciate if you could send them to me, so we can link them here...
